apps/nextjs-website/src/_contents/appIo/appIo.ts

```diff
@@ -4,12 +4,14 @@ import { appIOGuideListsPath } from '@/_contents/appIo/guideListsPath';
 import { appIoOverviewPath } from '@/_contents/appIo/overviewPath';
 import { appIoQuickStartGuidePath } from '@/_contents/appIo/quickStartGuidePath';
 import { appIoTutorialListsPath } from '@/_contents/appIo/tutorialListsPath';
+import { baseUrl } from '@/config';
 
 export const appIo: Product = {
   name: "L'App IO",
   description:
     'Raccogli tutti i servizi digitali del tuo ente in un’unica piattaforma e interagisci in modo semplice e sicuro con i cittadini.',
   svgPath: '/icons/appIo.svg',
+  pngPath: `${baseUrl}/icons/appIo.png`,
```

Comment:
It's not clear to me why a path contains the `baseUrl` and the other doesn't.

Reply:
The svgPath is only for internal use, the pngPath is used on the og meta tag which requires an absolute URL. We should rename it as `pngUrl`

Reply:
It's still used in the cards of the Product Showcase: `apps/nextjs-website/src/components/organisms/ProductsShowcase/ProductsShowcase.tsx`. We cannot remove it!